docs/source/release-process.rst

```diff
@@ -126,23 +126,20 @@ Edit your ``~/.pypirc`` file to look like:
    [distutils]
    index-servers =
        pypi
-       pypitest
 
    [pypi]
    username:
    password:
 
-   [pypitest]
-   repository: https://testpypi.python.org/pypi
-   username:
-   password:
-
 after that, run:
 
 .. code-block:: bash
 
-   $ python setup.py sdist upload -r pypitest
+   $ pip install twine
+   $ twine upload --repository-url https://test.pypi.org/legacy/ dist/zipline-<version-number>.tar.gz
```

Comment:
given this, we don't need to edit the pypirc at all

Reply:
You still need it if you're going to include pypi creds there for the non-test upload I think?

Reply:
it just prompts you